rest_api_flint.example/Dockerfile

```diff
@@ -10,8 +10,7 @@ RUN set -xe \
 RUN pip install --no-cache-dir --upgrade pip
 COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
-ENV PORT 8080
 
 COPY . .
 
-CMD ["exec", "gunicorn", "--bind", ":$PORT", "--workers", "1", "--threads", "8", "--timeout", "0", "app:app"]
+CMD ["gunicorn", "--bind", ":$PORT", "--workers", "1", "--threads", "8", "--timeout", "0", "app:app"]
```

Comment:
This will return an error. Since you removed the `PORT 8080` earlier this would not be valid. Instead, change `:$PORT` to `0.0.0.0:8080` and it should work in best case.